### ISSUE TITLE
Can no longer activate workflow

### ISSUE THREAD

**Issue by madchap** [n8n-io/n8n#1715]
**Describe the bug**
After updating docker container from 0.111.0 to 0.117.0, I can no longer activate my workflow. Apparently, the `save` works.

The error is

```
There was a problem and the workflow could not be activated:
Your request is invalid or could not be processed by the service
```

Console shows:

![image](https://user-images.githubusercontent.com/5468769/116320699-fcadc600-a7b8-11eb-9262-5e20d2212dd0.png)

**To Reproduce**
- I just updated, so unsure. I have a single job on that instance.

**Expected behavior**
I can activate the workflow.

**Environment (please complete the following information):**
Tried 0.116.1 and 0.117.0.

I didn't see any breaking changes related to this.

**Comment by madchap** [comment]
I moved back to 0.111.0 and the workflow activates.

**Comment by janober** [comment]
Can you please share your workflow so that we can try to reproduce the problem. Thanks.

**Comment by madchap** [comment]
[Right here](https://gist.github.com/madchap/34e6d9c4c3bea521f02e77cc1dc671a9), downloaded from [0.111.0.] Thanks.

**Comment by janober** [comment]
Thanks a lot @madchap. That was very helpful. I could now identify the problem. It had to do with the way we handle errors. That got changed recently and had that unintended side-effect on three nodes (one being the GitHub Trigger Node).

Just fixed the bug and will release it with the next version.

**Comment by janober** [comment]
Got released with n8n@0.118.0
